### ISSUE TITLE
TWE: Template Issue: Research: Interview Transcripts: Clean up and De-identify

### ISSUE THREAD

**Issue by pandanista** [hackforla/internship#509]
### Overview
We need to create a template issue for `TWE: Research: Interview Transcripts: Clean up and De-identify` so that we have a consistent, transparent, defined, and self-supporting structure for people creating them.

#### Customize this issue
 - Customize this issue by adding the Name of Issue Template. 
    - _Note: An issue template's name is different from the issue title created from the template._
    - [x] Search for `TWE: Research: Interview Transcripts: Clean up and De-identify` by using the browser search (**Mac**: ⌘+F; **PC**: Ctrl+F)
    - [x] Replace the keyword string (including text and the brackets) below with the name of your issue template according to the template naming convention: `TWE: Role: Direct object: Action verb`
        - The role is the primary owner of the issue template (e.g., Research, Design, Product)
        - Direct object should be the item that we're updating or working on (e.g., Wiki, Interviews, Roadmap)
        - Action verb should be the main descriptor of the action items (e.g., Update, Create, Transcribe)

#### Customize the issue settings (located in right sidebar)
Change the issue settings to ensure the issue is properly categorized and easy to manage.
- [x] Under Assignees, assign yourself.
- [x] Under Projects, choose gear, choose Repository, and choose TWE Project Management.
- [x] Add the milestone: Project Management
- [x] Add a feature label specific to what the template is related to, if available.  If not clear what you should use, ask product.

#### Customize template draft
Create a draft of new template, which will be known as our "draft".
- [x] Copy resource 1.01 Content to copy for draft to a new comment on this issue.
- [x] In your draft, fill out the top part (called front matter) with the following information.  See resource 1.02 Front Matter Resources section below for definitions and examples.
    - [x] Name
    - [x] About
    - [x] Title
    - [x] Labels
- [x] Define any Dependencies (or remove the dependency section if there is no dependency)
- [x] Customize the Overview section
- [x] Fill out the Action Items section (in your draft) by defining the steps that are required to be done, keeping the following in mind:
   - [x] The person working on this issue might be new to the org, so the issue should be able to be worked on without any prior knowledge
   - [x] If the person needs to add links, or put documents in specific locations, the place where they can do that action must be linked in the resources section
      - [x] The links in the resources section provided by the issue creator should be in the same order as the Action Items 
      - [x] Refer to resource 1.03 Example Action Items for Finding Wiki Page and Template for an example of how to write clear instructions that avoid compound steps (steps within a step).
   - [x] Number each resource in the order each resource appears in the Action Items
       - [x] Group resources into numbers and decimals (e.g., Resource 1.01, Resource 2.01)
       - [x] Group the resources based on whether they are available when making the issues or they need to be created from this issue
           - If the resource is available, then group it under the **Resources for creating this issue** section (e.g., Resource 1.01, Resource 1.02, Resource 1.03)
           - If the resource will be delivered or created from this issue, then group it under the **Resource links gathered during the making of this issue** section (e.g., Resource 2.01, Resource 2.02, Resource 2.03)
- [x] If documents or folders (sheets, docs, presentations, dashboard, etc.) will be created, find the document template that should be used by checking resource 1.04 WIKI Templates by role pages.  
  - [ ] If there is a template
     - [ ] Add a link to the template into the ` #### Resources for creating this issue`  section of your draft
  - [x] If there is no template yet
    - [x] Put a placeholder into the` #### Resources for creating this issue`  section of the draft `[TWE: Template Issue: [Role]: [Replace ACTION VERB][Replace DIRECT OBJECT]`
- [x] Get sign-off by Product on the draft

#### If this is for a template draft that needs testing at a later stage, please use the following handoff steps:
- [x] Add `feature: template creation (step 1)` to this issue
- [x] Once the template draft is finished, assign a team member or yourself to create testing issues 
- [x] Once the testings issues are completed, get sign-off by Product on the template draft
- [x] Create or update the template issue on the repo 
- [x] Add the link to the new issue template once created to resource 2.01.  Find by clicking on the **Get started** button to the right of the new template listed on [new issue page](https://github.com/hackforla/internship/issues/new/choose)
- [x] Update the appropriate place in the wiki for your template (see resources section)

#### Resources for creating this issue
- 1.01 Content to copy for draft
```
Draft of template issue for `[Replace NAME OF ISSUE TEMPLATE]`
---
**name:** TWE: Template Issue: [Replace ROLE]: [Replace ACTION VERB][Replace DIRECT OBJECT]
**about:** This issue helps you structure your template issue to be consistent with our other template issues in [Replace DIRECT OBJECT]
** title:** TWE: [Replace ROLE]: [Replace DIRECT OBJECT]: [Replace ACTION VERB]
**labels:** 'feature: board / google drive / tools, feature: missing, milestone: missing, role: missing, size: missing', 'Participant Type: missing', 'complexity: missing'
**assignees:**

---
#### Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- Customize the issue
    - [ ] Copy and paste entire template text in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
    - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
    - [ ] Search for `[Replace`
    - [ ] Replace with whatever the keyword or information is asking for.
- Customize the issue settings (located in right sidebar)
    - [ ] Under Labels, add labels for any labels identified as missing
    - [ ] Remove the corresponding missing labels
    - [ ] Under Projects, choose gear, choose "Repository," and choose TWE: Project Board.
    - [ ] Add the milestone: [Replace MILESTONE]
    - [ ] Add a feature label specific to what the template is related to, if available. If not clear what you should use, ask product.
- Customize dependencies 
    - [ ] If no dependencies, delete dependency section
    - [ ] If dependencies, add the dependencies (any issue or reason that would prevent the action items from being completed)
- Remove this section.
    - [ ] Delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
We need to [Replace ACTION VERB] [Replace DIRECT OBJECT] for [Replace REASON].

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, coding, etc.

### Resources
#### Resources for creating this issue
- 1.01
- 1.02

#### Resources gathered during the completion of this issue
- 2.01
- 2.02
```

- 1.02 Front Matter Resources
   <details>
   <summary>Front Matter Fields and Meaning</summary>

   - name: (name that appear on the new issue tab)
   - about: (is the description that appears next to the issue template on the new issue tab)
   - title: (the title that appears when the issue is created.  User then customizes this title for their issue)
       - Use the naming convention: TWE: Role: Direct object: Action verb
       - If more than one word should go into the direct object spot, use multiple square brackets with the keyword (see example below)
   - labels: (choose the labels that will be the same each time, and choose missing labels for the ones that depend on content)
   - assignees:  
   </details>
   <details>
   <summary>Front Matter Field Example</summary>

   - name: 'TWE: Template Issue: Research: Create Roadmap'
   - about:  This issue helps you structure your template issue to be consistent with our other template issues in roadmap
   - title: 'TWE: Research: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF RESEARCH METHOD]: Roadmap : Create'
   - labels: 'feature: research', 'feature: roadmap', 'milestone: missing', 'role: UI/UX research', 'Participant Type: missing', 'Research 000: missing', 'complexity: missing'
   - 'size: 0.50pt'
   - assignees: ''
   </details>

- 1.03 Example Action Items for Finding Wiki Page and Template
   <details>
   <summary>Example Action Items</summary>

   ```
   ### Action Items
   - Customize wiki page
        - [ ] Go to link 1.01 Internship Wiki, Join the team page
        - [ ] Find wiki page for this issue by looking at role pages.
        - [ ] Copy the link for the wiki page.
        - [ ] Add the link to Resource 2.01
           - [ ] Add the name of the link in square brackets
           - [ ] Add the URL in parenthesis without a space between the square bracket and parenthesis
   - Customize document template.
       - [ ] Go to link 1.02 Design Process Overview
       - [ ] Find document template that should be used in this issue.
       - [ ] Copy the link for the template.
       - [ ] Add the link to Resource 2.02
           - [ ] Add the name of the link in square brackets
           - [ ] Add the URL in parenthesis without a space between the square bracket and parenthesis
   ```

   </details>

   <details>
   <summary>Example Resource Lists</summary>

   ```
   ### Resources for creating this issue
   - 1.01. [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team)
   - 1.02. [Design Process Overview](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

   ### Resource links gathered during the making of this issue
   - 2.01. [Wiki page for [Replace TYPE OF ROLE]]
   - 2.02. [TWE: Template: [Replace NAME OF TEMPLATE]]
   ```

   </details>

- 1.04 WIKI Templates by role pages
   - [TWE: WIKI: UX Research Guides Templates](https://github.com/hackforla/internship/wiki/UX-Research-Guides-Templates) 
   - [Design](https://github.com/hackforla/internship/wiki/Design-Process-Overview)
   - [Product](https://github.com/hackforla/internship/wiki/Product)
   - [Business Analyst / Data Science]
   - [Software Engineer]
   - [Marketing]
   - default document templates, in case you don't have one yet, can be found on the [Design Process Overview page](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

#### Resources gathered during the completion of this issue
- 2.01 [TWE: Research: Interview Transcripts: Clean Up & De-identify](https://github.com/hackforla/internship/issues/new?template=research--clean-up-and-de-identify-interview-transcripts.md#issuecomment-2594623389)

**Comment by pandanista** [comment]
# Draft of template issue for cleaning up and de-identifying interview transcripts 

**name:** Research: Cleaning Up and De-identifying Interview Transcripts  
**about:** The process to clean up and de-identify interview transcripts
**title:** TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings: De-identify: [Replace TYPE OF PARTICIPANT] [Replace PARTICIPANT NUMBER]
**labels:** 'feature: recording transcription', 'feature: research', 'Participant Type: missing', 'Research 000: missing', 'size: 1pt', 'role: UI/UX research', 'milestone: missing'
**assignees:** 

---
#### Instructions for creating this issue
The person who creates this issue should use links in the section "Resources for creating this issue" (i.e., 1.01, 1.02, etc.) to customize this issue.
- Customize the issue
    - [ ] Open the "Research Overview" wiki page (Resource 1.01)
    - [ ] Choose the appropriate research roadmap or research plan folder (ie., RP###) on the wiki page 
    - Use the information on the wiki page and in the research roadmap or research plan to customize this issue
    - [ ] Copy and paste entire template in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
    - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
       - Year 
           - [ ] Find/Replace `[Replace YY]` with research year (Two digit format, ex: "22" for year "2022") 
       - Research Plan Number
            - [ ] Find/Replace `[Replace 000]` with the research plan number (Three digit format, ex: "001" for research plan "RP001") 
       - Type of Participant
           - [ ] Find/Replace `[Replace TYPE OF PARTICIPANT]` with the participant type (ex: "Intern" or "Mentor")  
       - Type of Interview
           - [ ] Find/Replace `[Replace TYPE OF INTERVIEW]` with the type of interview (ex: "Intake" or "Exit")
       - Participant Number
           - [ ] Find/Replace `[Replace PARTICIPANT NUMBER]` with the number assigned to the participant (Three digit format, ex: "001", and can be found in RP[Replace 000]'s Roll Call and Session Table) 
    - [ ] Copy and paste the edited template from text editor back into this Github Issue, replacing all original text
    - [ ] Re-check all the checkboxes above after copying and pasting the edited template
    - [ ] Close the text editor
    - Update the issue title
        - [ ] Copy 
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings: Transcribe: [Replace TYPE OF PARTICIPANT] [Replace PARTICIPANT NUMBER]
        ``` 
         - [ ] Choose "Edit" in the title section
        - [ ] Paste the text you copied into the issue title
        - [ ] Choose "Save"
- Customize the issue settings (located in right sidebar)
    - [ ] Under Assignees, assign a Researcher if it has been determined. Otherwise, leave it blank.
    - [ ] Under Labels,  choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Research: RP[Replace 000]`
    - [ ] Under Labels, choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Participant Type: [Replace TYPE OF PARTICIPANT]`
    - [ ] Remove the `Research 000: missing` and `Participant Type: missing` labels
    - [ ] Under Projects, choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), choose `Repository`, and choose `Program Management`
    - [ ] Add the milestone: choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and choose `Research Analysis`
    - [ ] Remove the label `milestone: missing`
- Customize dependencies 
    - [ ] If you know of dependencies, add them below (a dependency is any issue or reason that would prevent the action items from being completed)
- Remove this section
    - [ ] After all the checkboxes above are complete and the issue is ready to be worked on, delete all text below the "Draft of template issue..." (top heading) and above "Overview" (heading below) -- i.e., the issue assignee will never see this part

### Dependencies  
- [ ] All the user interview issues have been created 
- [ ] Interview Recording Upload Form needs to be created 
- [ ] Interview recordings have been uploaded and moved into Internship - PII's "My Drive"
- [ ] Interview transcripts have been created

### Overview 
We need to clean up and de-identify RP[Replace 000]'s [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] interview transcripts so that we can move on to the data analysis and insights generation phase. 

During these tasks, you will be asked to add links to the Resources/Instructions section, and that section is at the bottom of this issue (go view it now).

Another tip is to use split browser windows side by side when working on any GitHub issue, so you can edit the issue on one window, and check the resources or click through links on the other window to avoid a lot of back and forth. Make sure to refresh both windows to reflect any changes you made, including updating the issue or checking the boxes on one of them.


### Action Items
- [x]  UX lead adds the assignee to the Internship - PII's "My Drive" as a Contributor so that they have access to internship - PII's My Drive where the recordings and transcripts with PII are stored.
   - [x] UX lead accesses the Internship - PII's "My Drive"
   - [x] Choose "Manage members", which is located towards the top right side of the browser <img width="200" alt="Screenshot 2023-08-09 at 6 09 36 PM" src="https://github.com/hackforla/internship/assets/57029070/f7cc7fbe-1b00-4ded-8b44-b10fdbccae46">
   - [x] Enter the assignee's email address and select the role as "Contributor" <img width="300" alt="Screenshot 2023-08-09 at 6 11 40 PM" src="https://github.com/hackforla/internship/assets/57029070/41ab3531-75b1-435a-836a-1b24be3ecb69">
   - [x] Confirm with the assignee that they have access to the Internship - PII's "My Drive" 


#### Customize Resource Links
- Customize Resource for Wiki Page Link
    - [ ] Go to the wiki page: Research Output Overview (Resources # 1.01)
    - [ ] Find relevant wiki page for RP[Replace 000]
    - [ ] Copy the link for the wiki page
    - [ ] In Resources # 2.01, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked <img width="100" alt="Screenshot 2023-02-21 at 6 47 15 PM" src="https://user-images.githubusercontent.com/57029070/220508425-5d0c0a1b-ef87-4ef2-a683-d4a21d5b2672.png">

- Customize Resource for this Research Plan's Google Drive Folder
    - [ ] Open the Google Drive's Research by Type Folder (Resources # 1.02)
    - [ ] Choose the `[Replace TYPE OF PARTICIPANT]` folder
    - [ ] Copy the link of the `[Replace TYPE OF PARTICIPANT]` folder
        1. Choose the three vertical dots on the right side of the `[Replace TYPE OF PARTICIPANT]` folder  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [ ] In Resources # 2.02, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

- Customize Resource for Interview Recordings and Transcriptions Tracking Sheet
    - [ ] Open the Research Plan's Google Drive folder in the Internship's shared drive (Resource # 2.02)
    - [ ] Locate the `TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings and Transcriptions Tracking Sheet` in the folder
    - [ ] Copy the link of `TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings and Transcriptions Tracking Sheet`
        1. Choose the three vertical dots on the top right side of the file  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [ ] In Resources # 2.03, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

- Customize Resource for Interview Recording Folder stored in the Internship - PII's `My Drive`
    - [ ] Log into your Google account so you will be able to access the Internship - PII's 'My Drive' in the next steps
    - [ ] Choose Shared drives > Internship - PII > Internship -PII
    - [ ] Choose `TWE Interview Recordings by Plan #` folder in Internship - PII's `My drive` (Resources # 1.03) 
    - [ ] Locate the video recording folder for RP[Replace 000] inside the folder `TWE Interview Recordings by Plan #`
    - [ ] Copy the link of the video recording folder for RP[Replace 000] 
        1. Choose the three vertical dots on the right side of the folder  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [ ] In Resources # 2.04, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
 
- Customize Resource for the De-identified Transcripts Folder in the Shared Drive 
    - [ ] Go to RP[Replace 000] folder (Resources # 2.02)
    - [ ] Locate the `RP[Replace 000] De-identified Transcripts` folder
    - [ ] Copy the link of the folder
        1. Choose the three vertical dots on the right side of the folder <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [ ] In Resources # 2.05, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in GitHub and make sure all the checkboxes above have been checked

- Customize Resources for the Recording and Transcript you are Assigned to
    - [ ] Check the title of this issue to identify the participant number, which comes after `De-identify:`
    - [ ] Open the `TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings and Transcriptions Tracking Sheet` (Resources # 2.03)
    - [ ] Locate the recording video in .mp4 format that matches the participant number in Column B of the tracking sheet
    - [ ] Copy the link of the recording that matches the participant number in Column B of the tracking sheet
    - [ ] In Resources # 2.06, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Locate the transcript in .txt format that matches the participant number in Column C of the tracking sheet
    - [ ] Copy the link of the .txt file that matches the participant number in Column C of the tracking sheet
    - [ ] In Resources # 2.07, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Prepare for cleaning up and de-identification by converting the .txt file into a Google Doc
        1. Click on the link in Column C `Transcription Link` that matches the participant number (Resources # 2.07)
        1. Choose `Open with Google Docs` [![Screenshot 2023-01-06 at 11 04 38 AM](https://user-images.githubusercontent.com/57029070/211081813-b49a22f4-1de2-4a5f-a10a-e7c7686ae1d0.png)](https://user-images.githubusercontent.com/57029070/211081813-b49a22f4-1de2-4a5f-a10a-e7c7686ae1d0.png)
        1. A Google Doc is generated in a new window with the same .txt file name 
        1. The Google Doc is now saved into the same folder with the corresponding video and .txt file
        1. Open the RP[Replace 000] Interview Recording folder (Resources # 2.04)
        1. Open the corresponding folder for [Replace TYPE OF PARTICIPANT] [Replace PARTICIPANT NUMBER] and check to make sure a Google Doc transcript has been created
   - [ ] Make a copy of the Google Doc that was just generated and rename it
        1. Choose the three vertical dots on the top right side of the file <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Make a copy"
        1. A copy of the Google Doc is generated in the same folder       
        1. Copy 
        ```
        To be de-identified-
        ``` 
        1. Go back to the folder where a copy of the Google doc was created
        1. Choose the three vertical dots on the top right side of the file <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Rename"
        1. Remove `Copy of` from the file name when the Rename text box pops up
        1. Paste what you copied into the beginning of the file name text box
        1.  Choose "Ok" 
    - [ ] Copy the link of the "to be de-identified" Google Doc transcript you just created
        1. Choose the three vertical dots on the top right side of the file <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [ ] In Resources # 2.08, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked


#### Clean up and de-identify the transcript
- [ ] Open the Google Doc transcript if it is not open yet (Resource # 2.08)
- [ ] Listen to the recording to get yourself familiar with the transcript
- [ ] Listen to the recording again and clean up the inaccuracies in the transcript as you read along
    - [ ] Separate the texts based on speakers and timestamps (timestamps should match the video file timestamps in case a researcher needs to go back and double-check the original video)
    - [ ] Add any missed words
    - [ ] Edit any typos and misidentified words
    - [ ] Deleted repeated words
    - [ ] Remove verbal filler words such as "um" and "uh" if they don't serve any purpose
        - Sometimes the verbal fillers can indicate the interviewee's emotions or thoughts. In this case, provide context in square brackets to demonstrate the emotions or thoughts (e.g., [The participant hesitated for a while before coming up with the answer]).
    - [ ] Where needed, add context in square brackets so that a reader could understand what was happening without watching the video (e.g., [steps off camera])
- Read through the transcripts again and make edits to de-identify any personal information
    - [ ] Search for the participant's name 
    - [ ] Replace with their participant number, i.e. I001
    - [ ] Search for unknown speakers in the transcript
    - [ ] Replace unknown speaker with UXR number or other participant type/number
    - [ ] Read through the transcripts to search for any identifiable information in the interview
    - [ ] Replace any personal identifiable information with non-identifiable terms. For example, if someone's name is mentioned, we should use their roles inside square brackets so the reader knows this information was redacted (e.g., [team member], [peer], [mentor], [manager]). If their specific school is mentioned, we should redact that info with either [high school] or [college].
    - [ ] If they mentioned any specific issues they worked on, make sure to remove the issue numbers and rephrase the issues they worked on
- Read through the edited transcripts again
    - [ ] Focus on punctuation, readability, and formatting.
- When you're satisfied that the transcript is completely deidentified
    - [ ] Click on the version history and select the most current version
    - [ ] Click the three dots <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96"> and select "Make a copy", and the "Copy version" window pops up
    - [ ] In the "Name" text box, remove the "Copy of" text at the beginning and the timestamp at the end from the file name
    - [ ] Copy 
    ```
    De-identified Interview Transcript-
   ```
    - [ ] Paste the text you just copied in front of the file name in the "Name" text box
        - The new file name should be formatted like "De-identified Interview Transcript-RP[Replace 000]-UX Researcher Abbreviation###-[Replace TYPE OF PARTICIPANT] Abbreviation###"
        - An example: "De-identified Interview Transcript-RP006-U007-I001" 
    - [ ] Save the renamed de-identified Google doc transcript into the `RP[Replace 000] De-identified Transcripts` folder. This is to make sure that the version history does not include any identifiable information.
        1. Click the folder icon under "Folder"
        1. Choose "All locations"
        1. Choose Shared drives > Internship > Internships > Research > Research by Type > [Replace TYPE OF PARTICIPANT] > folder for RP[Replace 000] > RP[Replace 000] De-identified Transcripts
        1. Choose "Select"
        1. Select "Make a copy"
        1. The copy of the file is generated in a new browser
        1. Dismiss the notification that says the file is now in the shared drive
    - [ ] Copy the link of the de-identified transcript in the shared drive
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [ ] Paste what you copied into the matching participant's cell in Column D `Transcription Link - de-identified` in the tracking sheet (Resources # 2.03)
    - [ ] In Resources # 2.09, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in GitHub and make sure all the checkboxes above have been checked

 
#### 
- [ ] Review with UX Lead
- [ ] Product sign-off
- [ ] UXR Lead or PM removes the assignee from the Internship - PII's My Drive when closing the issue if the assignee no longer needs to access the PII drive.


### Resources/Instructions
#### Resources for creating this issue
1.01 [Wiki: Research Output Overview](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
1.02 [Google Drive Folder: Research by Type](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=share_link)
1.03* [TWE Interview Recordings by Plan #](https://drive.google.com/drive/folders/1lvP_k4qGKD05QzbHBtIpEl4B3X7ylzoh?usp=sharing)

#### Resources gathered during the completion of this issue
2.01 [Wiki: Research Plan: RP[Replace 000]]
2.02 [Google Drive Folder: RP[Replace 000]
2.03 [TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings and Transcriptions Tracking Sheet]
2.04* [RP[Replace 000] Interview Recording Folder]
2.05 [RP[Replace 000] De-identified Transcripts Folder]
2.06 [[Replace TYPE OF PARTICIPANT] [Replace PARTICIPANT NUMBER] Recording Link]
2.07 [[Replace TYPE OF PARTICIPANT] [Replace PARTICIPANT NUMBER] Transcript .txt File] 
2.08 [[Replace TYPE OF PARTICIPANT] [Replace PARTICIPANT NUMBER] To be De-identified Transcript Google Doc]
2.09 [[Replace TYPE OF PARTICIPANT] [Replace PARTICIPANT NUMBER] De-identified Transcript Google Doc]

*This folder can only be accessed from the Internship - PII's "My Drive"

**Comment by pandanista** [comment]
I've updated the template based on feedback in issue #478. This template is ready to be tested after the issue template testing is done. The issue template is #508, and the testing is #535.

**Comment by ExperimentsInHonesty** [comment]
@pandanista once the testing template is deployed please create an issue to test a template and add the label `status: template testing (step 2b)`

**Comment by pandanista** [comment]
[This template](https://github.com/hackforla/internship/issues/509#issuecomment-1789691769) might need to add
- some of the steps in creating the participants list and the de-identified participant list 
    - Example of the steps can be found in [this comment](https://github.com/hackforla/internship/issues/481#issuecomment-2138505878)
- a peer review stage
    - Example of the steps can be found in [this comment ](https://github.com/hackforla/internship/issues/481#issuecomment-2138503293)

**Comment by pandanista** [comment]
## A peer review for cleaning up and de-identifying transcript draft:

We need a round of review on [[Replace PARTICIPANT NUMBER] De-identified Transcript](INSERT FILE URL). Thank you.

### Action Items for Peer Review
- [x] Review this issue
     - [x] Read the issue overview to learn about why this document was created.
     - [x] Locate draft of the document in the Resources.
- [x] Review resources in 1.04 for easy reference on how to clean up the interview transcripts (those resources are listed below for easier access)
    - [x] 1.04 [Guidelines to Interviews Page 8](https://drive.google.com/open?id=19580_-BXfHPC9gTcAj35O2nzAtdNwTxp&disco=AAAA_U2KB_A)
        - [x] 1.04.01 [Brackets vs. Parentheses](https://www.strose.edu/wp-content/uploads/2015/10/Brackets-Handout-Cynthia2.pdf)
        - [x] 1.04.02 [Video folder](https://drive.google.com/drive/folders/1WPzYSrhWYluIqigjsHKpSYIhpQf7XcM5)
        - [x] 1.04.03 [Cleaning Up Zoom Transcriptions for Qualitative Research](https://crdpala.org/2022/10/05/cleaning-up-zoom-transcriptions-for-qualitative-research/)
        - [x] 1.04.04 [Determining Best Practice for Filler Words in Captions and Transcripts](https://blogs.vcu.edu/birdlab/2020/01/09/determining-best-practice-for-filler-words-in-captions-and-transcripts/)
- [x] Review the de-identified transcript
     - [x] Preview the draft quickly to get a sense of how the draft is structured and the type of content.
     - [x] Review the structure of the draft to ensure that the headings (in a Google Doc) or columns (Google Sheet) are clearly identified. 
     - [x] Install [LanguageTool Chrome Extension](https://chrome.google.com/webstore/detail/grammar-checker-paraphras/oldceeleldhonbafppcapldpdifcinji) to clean up the punctuation and verbal ticks in the transcript.
     - [x] A few guidelines on the transcript formatting:
         1. Use the 'in-line' format as seen [here in Intern 001's transcript](https://docs.google.com/document/d/1C8-qEAot7Sw5rhpfuplP0D0SLxypdChUigC1u7LCWJA/edit)
         1. Use 6 digit format for all the timestamps, for example, 01:01:01 as hour one/minute one/second one
         1. Use "Interviewer" and "Intern ###" throughout the transcript. See an example [here in Intern 001's transcript](https://docs.google.com/document/d/1C8-qEAot7Sw5rhpfuplP0D0SLxypdChUigC1u7LCWJA/edit).
            - If there were more than two people recorded on the transcript, name the third person based on the interview set-up or context, such as Program Manager, Note Taker, etc. If you are not sure what to name them, ask leads on the project to clarify.
          1. If there are other numerical values in the transcript, transcribe them using texts. For example, if a time such as 6:30 is mentioned, type out six thirty when cleaning up the transcript. Another example is if the interviewee mentioned he mainly sought help from 3 other peers, type out "three" instead of using the number "3".
- [x] Provide feedback
     - [x] Add feedback in the form of (1) questions and/or (2) explicit instructions for how the content should be improved. Feedback in a Google Doc should be given in the form of document comments. 
     - [x] If there are edits to be made, please use "suggesting mode" in Google doc instead of "editing mode".
     - [x] Determine the accuracy of the content for each section (Google doc). If you find an inaccuracy, error, or mis-categorization, explain how the information presented could be improved.
     - [x] For each suggested improvement, provide a link or reference within your comment to justify your suggestion (if available).
- [x] Once all the steps have been completed, reach out to the UX Lead to determine if they can complete the next steps in the issue.

**Comment by pandanista** [comment]
- Updated the template by adding the participants list steps.
- Saved [this comment](https://github.com/hackforla/internship/issues/509#issuecomment-2403665438) as a saved reply.

**Comment by pandanista** [comment]
working on final edits on the template draft.

**Comment by pandanista** [comment]
- The testing issue has been created and assigned for testing.
    - #682
- Added step 2b label to this issue, and moved this issue to the review column so we can incorporate any feedback needed resulting from the testing.

**Comment by pandanista** [comment]
- The [template](https://github.com/hackforla/internship/issues/509#issuecomment-1789691769) has been updated.
- The [peer review saved reply](https://github.com/hackforla/internship/issues/509#issuecomment-2403665438) has been updated.
- The template testing has been completed, and ready to be deployed.
- Assigned a team member to do a peer review using a saved reply.
    - Can a saved reply be saved into repo as well?

**Comment by pandanista** [comment]
Editing the peer review saved reply https://github.com/hackforla/internship/issues/509#issuecomment-2403665438.

**Comment by pandanista** [comment]
- The peer review is done. See https://github.com/hackforla/internship/issues/683#issuecomment-2597431522
- The peer review template has been created as [a wiki page](https://github.com/hackforla/internship/wiki/A-Peer-Review-for-Cleaning-Up-and-De%E2%80%90identifying-Transcripts) and linked under the issue templates section of [the UX research guides templates page](https://github.com/hackforla/internship/wiki/UX-Research-Guides-Templates)
